Packs/CommonScripts/Scripts/RepopulateFiles/RepopulateFiles.py

```diff
@@ -37,8 +39,11 @@ def main():
         Demisto entry.
     """
     entries = demisto.executeCommand('getEntries', {'filter': {'categories': ['attachments']}})
```

Comment:
please use `is_error` to check for error entry returned from `executeCommand` and `get_error` to retrieve it